File: Modul 4/Soal 4/PRAK304-2210817210006-Farlyhaydy H.Djalil.c
#include <stdio.h>

int main()
{
    int a, i=0;
    while(i<=4){
    scanf("%d", &a);
    if(a>=1 && a<=9)
    printf("Satuan\n\n");
    else if(a<1 && a>-50){
    printf("Nol\n\n");
    }else if(a>99){
    printf("Anda Menginput Melebihi Limit Bilangan\n\n");
    }else if(a>=20 && a<=99){
    printf("Puluhan\n\n");
    }else
    printf("Belasan");
    i++;
    }
}

File: Modul 4/Soal 4/PRAK304-2210817210006-Farlyhaydy H.Djalil.py
i=0
while True:
    i=i+1
    a=int(input())
    if (a>=1 and a<=9):
        print("Satuan\n")
    elif (a<1 and a>-50):
        print("Nol\n")
    elif (a>99):
        print("Anda Menginput Melebihi Limit Bilangan\n")
    elif (a>=20 and a<=99):
        print("Puluhan\n")
    else :
        print("Belasan")
    if (i>4): 
        break